src/ui/about_dialog.ui

```diff
@@ -42,7 +42,8 @@
   </property>
   <property name="windowIcon">
    <iconset resource="../mudlet.qrc">
-    <normaloff>:/icons/mudlet_information.png</normaloff>:/icons/mudlet_information.png</iconset>
+    <normaloff>:/icons/mudlet_information.png</normaloff>:/icons/mudlet_information.png
```

Comment:
That might be a defect - it is introducing a **line-feed** and **spaces** onto the end of the ***string*** that is a (in this case *resource*) file name...

:worried: